### PR TITLE
[Yang-models] Modify mirror yang model src_port to match CONFIG_DB

### PR DESCRIPTION
**Pull request by rupesh-k**
Fixes Mirror Yang model src_port to match CONFIG_DB

#### Why I did it
Fixes issue https://github.com/sonic-net/sonic-buildimage/issues/12397#issuecomment-1343109874

### CODE REVIEW

**Comment by zhangyanzhao** [comment]
@rupesh-k  can you please sign the EasyCLA, that is required before you can merge code. Thanks.

File: src/sonic-yang-models/yang-models/sonic-mirror-session.yang [hunk 1]
```diff
@@ -154,14 +154,9 @@ module sonic-mirror-session {
...
-                    type union {
-                        type leafref {
-                            path "/port:sonic-port/port:PORT/port:PORT_LIST/port:name";
-                        }
-                        type leafref {
-                            path "/lag:sonic-portchannel/lag:PORTCHANNEL/lag:PORTCHANNEL_LIST/lag:name";
-                        }
+                leaf src_port {
+                    type string {
+                        length 1..256;
```
Comment (ganglyu) [inline src/sonic-yang-models/yang-models/sonic-mirror-session.yang:159]: Is 256 enough for src_port?

Comment (rupesh-k) [reply src/sonic-yang-models/yang-models/sonic-mirror-session.yang]: modified to 2048. should allow configuring upto 200 ports in single session.

**Review by ganglyu — APPROVED** [review]

**Comment by zhangyanzhao** [comment]
@qiluo-msft  can you please double check and see if this PR can be merged? Thanks.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**